docs/cow-protocol/tutorials/cow-swap/swap-and-bridge.mdx

```diff
@@ -0,0 +1,198 @@
+import ActivitiesButtonImg from '/img/cowswap/bridging/activities-button.png'
+import BridgingInProgressImg from '/img/cowswap/bridging/bridging-in-progress.png'
+import BridgingSuccessImg from '/img/cowswap/bridging/bridging-success.png'
+import SwapSuccessImg from '/img/cowswap/bridging/swap-success.png'
+import AccountProxyCardImg from '/img/cowswap/bridging/account-proxy-card.png'
+import AccountProxyListImg from '/img/cowswap/bridging/account-proxy-list.png'
+import CoWExplorerImg from '/img/cowswap/bridging/cow-explorer.png'
+
+# Swap & Bridge
+
+The **Swap & Bridge** feature on [swap.cow.fi](https://swap.cow.fi) lets you swap a token on one network and receive a different token on another network — all in one seamless transaction.
+
+This guide will walk you through the process step-by-step, from setting up your trade to tracking it in the explorer.
+
+<img src={SwapFormImg.src} style={{ maxWidth: 500 }} alt="Swap and Bridge trade" />
+
+---
+
+## 1. Setting up a Swap with Different Network Assets
+
+When using **Swap & Bridge**, you’ll start by selecting the tokens you want to trade — just like a regular swap — but with the option to choose a different destination chain.
+
+### Source Token Selector
+
+Open a source token selector:
+
+<img src={SourceTokenSelectorPlaceImg.src} style={{ maxWidth: 500 }} alt="Open source token selector" />
+
+Select the network and token you want to sell:
+
+<img src={SourceTokenSelectorImg.src} style={{ maxWidth: 500 }} alt="Source token selector" />
+
+### Destination Token Selector
+
+Similarly to Source Token Selector, open the selector, choose a network and token you would like to buy
+
+<img src={DestinationTokenSelectorImg.src} style={{ maxWidth: 500 }} alt="Destination token selector" />
+
+:::tip
+
+The list of available destination networks depends on the source token you’ve selected.
+
+If you choose a destination network different from the source token’s network:
+- You will only see tokens supported by the bridge provider for that chain.
+
+If the destination network is the same as the source network:
+- You’ll see the regular list of tokens from the token lists.
+
+:::
+
+:::note
+
+**Tokens must be different.** Bridging the same token between two networks is not yet supported. This feature is under development.
+
+:::
+
+---
+
+## 2. Creating a Swap & Bridge Trade
+
+Once your tokens and networks are set, you can proceed to create the order.
+
+### Token Approval
+
+If it’s your first time selling this token through CoW Swap, you’ll need to approve it.
+This is the standard ERC-20 approval process that allows CoW Protocol to use your tokens for the trade.
+
+Approvals can happen through:
+
+* Regular ERC-20 allowance transactions
+* [EIP-6963](https://eips.ethereum.org/EIPS/eip-6963) permit flow (when supported)
+
+<img src={SignApprovalImg.src} alt="Sign an approval" />
+
+### Signing Steps
+
+1. **Sign the post-hook call** for the bridge provider.
```

Comment:
This section could be expanded a bit further.
Explain the order and what is being signed in the first request (the bridge calldata).

Reply:
Thanks!
Added more details:

<img width="995" height="345" alt="image" src="https://github.com/user-attachments/assets/1ae9ce6f-d643-442e-b9ba-c6890c82a98f" />